libraries/lib-datahandler/src/main/thrift/vulnerabilities.thrift

```diff
@@ -42,13 +49,22 @@ struct Vulnerability{
     15: optional string priority,
     16: optional string priorityText,
     17: optional string action,
-    19: optional string impact,
+    19: optional map<string, string> impact,
     20: optional string legalNotice,
     21: optional set<string> assignedExtComponentIds,
     22: optional set<CVEReference> cveReferences,
     23: optional set<VendorAdvisory> vendorAdvisories,
     24: optional string extendedDescription,
     25: optional set<string> references,
+
+    //additional from CVE earch data
+    30: optional double cvss,
+    31: optional bool isSetCvss,
```

Comment:
Why is this field necessary? Is the method `isSetCvss()` generated by thrift not enough? I'd say it definitely creates confusion.